File: server/src/main/java/io/github/eddiediamondfire/economyplus/utils/MessageManager.java
package io.github.eddiediamondfire.economyplus.utils;

import io.github.eddiediamondfire.economyplus.Main;
import org.bukkit.Bukkit;
import org.bukkit.ChatColor;
import org.bukkit.command.ConsoleCommandSender;
import org.bukkit.entity.Player;

public class MessageManager {

    private Main plugin = null;
    private boolean debugMode;
    public MessageManager(Main plugin)
    {
        this.plugin = plugin;
    }
    public static void sendMessage(Player player, ChatColor colour, String message){
        player.sendMessage(colour + message);
    }

    public static void sendMessage(ChatColor colour, String message){
        ConsoleCommandSender console = Bukkit.getConsoleSender();
        console.sendMessage(colour + message);
    }

    public static void sendDebugMessage(ChatColor colour, String message){

    }
}

File: server/src/main/kotlin/io/github/eddiediamondfire/economyplus/utils/MessageManager.kt
package io.github.eddiediamondfire.economyplus.utils

import io.github.eddiediamondfire.economyplus.Main
import org.bukkit.Bukkit
import org.bukkit.ChatColor
import org.bukkit.command.ConsoleCommandSender
import org.bukkit.entity.Player

class MessageManager constructor(pluginClass: Main, debugMode : Boolean) {

    private val plugin : Main = pluginClass

    private val debug : Boolean = debugMode

    companion object {
        @JvmStatic
        fun sendMessage(player: Player, colour: ChatColor, message: String){
            player.sendMessage("$colour $message");
        }

        @JvmStatic
        fun sendMessage(colour : ChatColor, message: String){
            val console: ConsoleCommandSender = Bukkit.getConsoleSender()
            console.sendMessage("$colour $message")
        }
    }

    fun sendDebugMessage(player: Player, colour: ChatColor, message: String)
    {
        if(debug){
            player.sendMessage("$colour $message")
        }
    }

    fun sendDebugMessage(colour: ChatColor, message: String)
    {
        val console: ConsoleCommandSender = Bukkit.getConsoleSender()
        if(debug){
            console.sendMessage("$colour $message")
        }
    }

}